fastlane/Fastfile

```diff
@@ -31,17 +31,32 @@ SUPPORTED_LOCALES = [
 platform :android do
   # Download the latest app translations from GlotPress and update the strings.xml files accordingly.
   #
-  lane :download_translations do
+  # @example Running the lane
+  #          bundle exec fastlane download_translations skip_commit:true
+  #
+  lane :download_translations do |skip_commit: false|
     android_download_translations(
       res_dir: RES_UI_DIR_PATH,
       glotpress_url: GLOTPRESS_UI_STRINGS_PROJECT_URL,
-      locales: SUPPORTED_LOCALES
+      locales: SUPPORTED_LOCALES,
+      skip_commit: true
     )
 
     android_download_translations(
       res_dir: RES_QUICK_EDITOR_DIR_PATH,
       glotpress_url: GLOTPRESS_QUICK_EDITOR_STRINGS_PROJECT_URL,
-      locales: SUPPORTED_LOCALES
+      locales: SUPPORTED_LOCALES,
+      skip_commit: true
+    )
+
+    next if skip_commit
+
+    strings_paths = [RES_UI_DIR_PATH, RES_QUICK_EDITOR_DIR_PATH]
+    git_add(path: strings_paths)
+    git_commit(
+      path: strings_paths,
+      message: 'Update translations',
+      allow_nothing_to_commit: true
     )
   end
```

Comment:
Alternate implementation idea (just being fancy here—though this could make the code easier to update if the projects ends up adding more modules—and thus more `strings.xml` files to translate—in the future?):

1. Replace the 4 constants at the top of the Fastfile (L7–L10) with
```ruby
GLOTPRESS_PROJECT_BASE_URL = 'https://translate.wordpress.com/projects/gravatar/gravatar-android-sdk'
RESOURCES_TO_TRANSLATE = {
  File.join('gravatar-ui', 'src', 'main', 'res'): "#{GLOTPRESS_UI_STRINGS_PROJECT_URL}/gravatar-ui/",
  File.join('gravatar-quickeditor', 'src', 'main', 'res'): "#{GLOTPRESS_UI_STRINGS_PROJECT_URL}/gravatar-quick-editor/"
}
```
2. Then use that to implement the rest of that lane in a loop:
```ruby
  RESOURCES_TO_TRANSLATE.each do |res_dir, gp_url|
    android_download_translations(
      res_dir: res_dir,
      glotpress_url: gp_url,
      locales: SUPPORTED_LOCALES,
      skip_commit: true
    )
  end

  next if skip_commit

  strings_paths = RESOURCES_TO_TRANSLATE.keys
  git_add(path: strings_paths)
  git_commit(
    path: strings_paths,
    message: 'Update translations',
    allow_nothing_to_commit: true
  )
```

Again, this is just me being fancy with Ruby; optional and not a blocker of course 😉

Reply:
Good idea, I'll take it -- perhaps Gravatar will remain as two modules but I think it makes this code more reusable / generic and doesn't complicate it a lot more.